tests/unit/h/services/group_members_test.py

```diff
@@ -99,22 +96,25 @@ def test_it_adds_users_in_userids(self, factories, group_members_service):
     def test_it_removes_members_not_present_in_userids(
         self, factories, group_members_service, creator
     ):
-        group = factories.Group(creator=creator)  # creator will be a member
-        new_members = [factories.User(), factories.User()]
-        group.members.append(new_members[0])
-        group.members.append(new_members[1])
+        group = factories.Group(
+            creator=creator,
+            memberships=[
+                GroupMembership(user=factories.User()),
+                GroupMembership(user=factories.User()),
+            ],
+        )
 
         group_members_service.update_members(group, [])
 
-        assert not group.members  # including the creator
```

Comment:
The comment is no longer applicable now that the factories no longer add creators as group members automatically.